lib/puppet/type/sensu_api_config.rb

```diff
@@ -1,3 +1,14 @@
+# @!puppet.type.param name
```

Comment:
Makes sense. Don't bother with the ones that are changing and we'll make another pass after the dust settles.